docs/api.yml

```diff
@@ -54,25 +54,18 @@ paths:
           content:
             application/json:
               example:
-                {
-                  'total': { 'value': 2, 'relation': 'eq' },
-                  'max_score': 0.5753642,
-                  'hits':
-                    [
-                      {
-                        '_index': 'documents',
-                        '_type': '_doc',
-                        '_id': 'PnJOIy',
-                        '_score': 0.5753642,
-                        '_source':
-                          {
-                            'documentId': 'PnJOIy',
-                            'firstName': 'toaster',
-                            'lastName': 'dude',
-                          },
-                      },
-                    ],
-                }
+                [
+                  {
+                    'documentId': '1',
+                    'index': 'documents',
+                    'metadata': { 'name': '123' },
+                  },
+                  {
+                    'documentId': '2',
+                    'index': 'documents',
+                    'metadata': { 'name': 'abc' },
+                  },
+                ]
```

Comment:
@t-davies would this make more sense

Reply:
maybe include the score too - could come in handy

Reply:
👌

Reply:
Yeah this looks good, happy with including the score too as Matt mentioned - just didn't want to tie us directly to the Elasticsearch responses. I do think we need a link to the document included as well though - unless we have a convention for fetching that instead - something like "https://evidence-store.hackney.gov.uk/documents/{documentId}" or something?

Reply:
Maybe?

Reply:
Can probably add something like the above as a placeholder for now

Reply:
So would we make another call in this route/use-case to get that url?

Reply:
If it's just a route on evidence store like "/documents/{documentId}" no need for an extra call here?

We can do some magic to do something like if you got o "/documents/{documentId}" it makes the call to S3 to get the object URL and then redirects you or something? That way we can defer actually calling S3 until the moment we need to. Happy to chat more if you like.